### PR TITLE
feature(drivers-service): fleets organization and members wip

### PR DESCRIPTION
**Pull request by afkzoro**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
This PR creates the Fleets feature.

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Comment by afkzoro** [comment]
@siradji suggestion on securing the endpoints?

File: apps/drivers-service/src/drivers-service.service.ts [hunk 1]
```diff
@@ -319,6 +328,122 @@ export class DriversServiceService {
...
+        firstName: payload.firstName,
+        lastName: payload.lastName,
+        phone: internationalisePhoneNumber(payload.phone),
+        email: payload.email,
+        password: await bcrypt.hash(payload.password, 10),
+        organization: createOrganization._id.toString()
+      })
+
+      await this.fleetOrgRepository.findOneAndUpdate({
+        _id: createOrganization._id
```
Comment (siradji) [inline apps/drivers-service/src/drivers-service.service.ts:359]: You need to convert the id to string using _id.toString()

File: apps/drivers-service/src/drivers-service.service.ts [hunk 2]
```diff
@@ -319,6 +328,122 @@ export class DriversServiceService {
...
+      }, {
+        $push: { members: newMember._id, owners: newMember._id }
```
Comment (siradji) [inline apps/drivers-service/src/drivers-service.service.ts:361]: Same as above. there will be a type cast error. All _ids should be converted go string

**Review by siradji — CHANGES_REQUESTED** [review]
Nice work. Fixed my comments and i will get it in

File: apps/drivers-service/src/drivers-service.service.ts (hunk 2 above)
Comment (afkzoro) [reply apps/drivers-service/src/drivers-service.service.ts]: Fixed @siradji

**Comment by siradji** [comment]
Lgtm

**Comment by siradji** [comment]
@afkzoro change the base pr branch to staging

**Comment by afkzoro** [comment]
Done

**Review by siradji — APPROVED** [review]

**Pull request merged**